packages/contracts-bedrock/scripts/outputs/DeleteOutput.s.sol

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { console } from "forge-std/console.sol";
+import { IMulticall3 } from "forge-std/interfaces/IMulticall3.sol";
+
+import { LibSort } from "../libraries/LibSort.sol";
+import { IGnosisSafe, Enum } from "../interfaces/IGnosisSafe.sol";
+import { SafeBuilder } from "../universal/SafeBuilder.sol";
+
+import { Types } from "../../contracts/libraries/Types.sol";
+import { FeeVault } from "../../contracts/universal/FeeVault.sol";
+import { L2OutputOracle } from "../../contracts/L1/L2OutputOracle.sol";
+import { Predeploys } from "../../contracts/libraries/Predeploys.sol";
+
+/**
+ * @title DeleteOutput
+ * @notice Deletes an output root from the L2OutputOracle.
+ * @notice Example usage is provided in the README documentation.
+ */
+contract DeleteOutput is SafeBuilder {
+    /**
+     * @notice A set of contract addresses for the script.
+     */
+    struct ContractSet {
+        address Safe;
+        address ProxyAdmin;
+        address L2OutputOracleProxy;
+    }
+
+    /**
+     * @notice A mapping of chainid to a ContractSet.
+     */
+    mapping(uint256 => ContractSet) internal _contracts;
+
+    /**
+     * @notice The l2 output index we will delete.
+     */
+    uint256 internal index;
+
+    /**
+     * @notice The address of the L2OutputOracle to target.
+     */
+    address internal oracle;
+
+    /**
+     * @notice Place the contract addresses in storage for ux.
+     */
+    function setUp() external {
+        _contracts[GOERLI] = ContractSet({
+            Safe: 0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f,
+            ProxyAdmin: 0x01d3670863c3F4b24D7b107900f0b75d4BbC6e0d,
+            L2OutputOracleProxy: 0xE6Dfba0953616Bacab0c9A8ecb3a9BBa77FC15c0
+        });
+    }
+
+    /**
+     * @notice Returns the ContractSet for the defined block chainid.
+     *
+     * @dev Reverts if no ContractSet is defined.
+     */
+    function contracts() public view returns (ContractSet memory) {
+        ContractSet memory cs = _contracts[block.chainid];
+        if (cs.Safe == address(0) && cs.ProxyAdmin == address(0) && cs.L2OutputOracleProxy == address(0)) {
```

Comment:
I feel like we want this to be an or condition instead of an and condition, so if any of them are unset